src/OpenStack.Net/OpenStack/Services/ContentDelivery/V1/ServiceRestrictionRule.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace OpenStack.Services.ContentDelivery.V1
+{
+    using Newtonsoft.Json;
+    using OpenStack.ObjectModel;
+
+    [JsonObject(MemberSerialization.OptIn)]
+    public class ServiceRestrictionRule : ExtensibleJsonObject
```

Comment:
:exclamation: This needs to be documented (along with several other elements in the same file).